src/components/Calculator.js

```diff
@@ -1,17 +1,28 @@
-import React from "react";
+import React from 'react';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-  }
+const Calculator = () => (
```

Comment:
- [ ] In this milestone, the component should be class-based, please change it back 😄

Reply:
@MahmoudBakr23 Hi, in this case, can you tell me how to bypass this error:  
`Line 3:1:  Component should be written as a pure function  react/prefer-stateless-function`
?
Thanks!

Reply:
@MahmoudBakr23 Because this way it does not compile

Reply:
@MahmoudBakr23  
Can you please tell me what to do with this?
![image](https://user-images.githubusercontent.com/103699974/199358366-465abb95-0e23-4590-9c08-ae9a298e66c2.png)